experimental/dds/tree2/src/test/snapshots/testTrees.ts

```diff
@@ -82,26 +106,162 @@ function generateTreeRecursively(
 	}
 }
 
-// TODO:
-// The implementation and usage of these implies the edit history of these trees is relevant, but its not exactly clear how.
-// More documentation on what kind of coverage over possible tree histories and contents this is supposed to provide is needed here.
-// Depending on the above, maybe these tests (or at least some of the cases) should probably be changed to use branches and not full trees to avoid depending on a Fluid runtime.
-// Currently these tests all replicate pre-attachment states.
-// Coverage for other states should be added (including cases with collaboration).
-export function generateTestTrees(): { name: string; tree: () => ISharedTree }[] {
-	return [
+export async function generateTestTrees(): Promise<
+	{
+		name: string;
+		summary: ISummaryTree;
+	}[]
+> {
+	const testTrees: {
+		only?: boolean;
+		skip?: boolean;
+		name: string;
+		tree: (a: (b: ISharedTree, validationName: string) => Promise<void>) => Promise<void>;
+	}[] = [
+		{
+			name: "move-across-fields",
+			tree: async (validate) => {
+				const provider = new TestTreeProviderLite(2);
+				const tree1 = provider.trees[0].view;
+				const tree2 = provider.trees[1].view;
+
+				const initialState: JsonableTree = {
```

Comment:
Maybe leave a note here that we're using older editing APIs and not editable-tree-2 because it didn't support cross-field move at the time of writing.